FogCreek/Glitch-Community
# Mirage power 1: replace ProjectsLoader with project context

## Links
* Remix link: https://mirage-power-1.glitch.me

## Changes:
* create context for managing project members
* make all project items fetch their own members (via context) only when visible
* all project items will now render both teams and users
* move add/remove user from team project logic to team editor 
* use project permissions, not users, for determining project options
* remove unused normalizeProjects from current user state

## How To Test:
* log into https://mirage-power-1.glitch.me
* verify that you can join and leave a project belonging to one of your teams, and the project's profile list updates when you do so
* verify that profile list renders for projects on user page, teams page, search results, recent projects header
* verify that profile list renders for project search results, e.g. in "Add Project" on a team's page
